samples/features/security/ledger/azure-sql-database/setup/azuredeploy.bicep

```diff
@@ -76,38 +79,68 @@ resource Server_Name_activeDirectory 'Microsoft.Sql/servers/administrators@2019-
   }
 }
 
-//////////////////////////////////////////////////////////////////////////////
-// Create the ContosoHR database using the DC-series hardware configuration //
-//////////////////////////////////////////////////////////////////////////////
-
-resource Database_Resource 'Microsoft.Sql/servers/databases@2020-08-01-preview' = {
-  name: '${Server_Name_resource.name}/ContosoHR'
+///////////////////////////////////
+// Create the ContosoHR database//
+//////////////////////////////////
+#disable-next-line BCP081
+resource Database_Resource 'Microsoft.Sql/servers/databases@2021-02-01-preview' = {
+  name: 'ContosoHR'
+  parent: Server_Name_resource
   location: location
-  tags: {}
   sku: {
-    name: 'GP_DC_2'
+    name: 'GP_S_Gen5_1'
```

Comment:
Is this the cheapest performance timer required for this sample?

Reply:
Yes, I picked this sku because it's serverless and only uses max 1 vCore.
When the DB is not used for demo, it is paused so the user only pays the storage which is cheap.